apps/builder/app/builder/features/project-settings/section-marketplace.tsx

```diff
@@ -116,6 +130,19 @@ const useMarketplaceApprovalStatus = () => {
           },
           handleSuccess
         );
+
+        // remove the deafult share link
+        load({ projectId: project.id }, (data) => {
+          const marketplaceLink = data.find(
+            (currentLink) => currentLink.name === marketplaceLinkName
```

Comment:
identifying an existing link by name would be very fragile

Reply:
since name is user defined, can be changed

Reply:
I think it should be possible to not delete anything, but instead create it once and just keep it hidden. Additionally maybe need to think about unpublishing too, so deletion might be still needed but when unpublished